common/client-libs/gateway-client/src/client.rs

```diff
@@ -33,14 +33,14 @@ use std::sync::Arc;
 use std::time::Duration;
 use tungstenite::protocol::Message;
 
-#[cfg(not(target_arch = "wasm32"))]
+#[cfg(unix)]
 use std::os::fd::RawFd;
 #[cfg(not(target_arch = "wasm32"))]
 use tokio::time::sleep;
 #[cfg(not(target_arch = "wasm32"))]
 use tokio_tungstenite::connect_async;
 
-#[cfg(target_arch = "wasm32")]
+#[cfg(not(unix))]
```

Comment:
so I guess we don't need the file descriptor in windows?

Reply:
Yep, I think they don't even have that notion, and this fd is needed only for android for now